src/lib/crypto/kimchi_backend/common/plonk_dlog_proof.ml

```diff
@@ -365,9 +365,7 @@ module Make (Inputs : Inputs_intf) = struct
     ; rot_selector
     ; lookup_aggregation
     ; lookup_table
-    ; lookup_sorted =
-        Array.init 5 ~f:(fun i ->
-            Option.try_with_join (fun () -> lookup_sorted.(i)) )
+    ; lookup_sorted = Vector.to_array lookup_sorted
```

Comment:
Would it make sense to have both `lookup_sorted` on line 279 have the same type as here? Meaning, either vectors or arrays in both cases.